riff-raff.yaml

```diff
@@ -14,13 +14,5 @@ deployments:
           name: image-url-signing-service-CODE
         PROD:
           filename: image-url-signing-service.zip
-          name: iimage-url-signing-service-PROD
-    dependencies:
-      - cloudformation
-  cloudformation:
-    type: cloud-formation
-    app: image-url-signing-service
-    parameters:
-      prependStackToCloudFormationStackName: true
-      cloudFormationStackName: image-url-signing-service
-      templatePath: cfn.yaml
+          name: image-url-signing-service-PROD
```

Comment:
Oh, I'm suprised to see these changes in here - are they needed for the cors change?

Reply:
We discussed this offline - there's an issue with RiffRaff deploying the cfn currently, so it's been removed for now.